datamol/convert.py

```diff
@@ -180,7 +180,61 @@ def to_inchi(mol: Union[str, Chem.rdchem.Mol]) -> Optional[str]:
     return Chem.MolToInchi(mol)
 
 
-def to_smarts(mol: Chem.rdchem.Mol) -> Optional[str]:
+def to_inchi_non_standard(
+    mol: Union[str, dm.Mol],
+    fixed_hydrogen_layer: bool = True,
+    undefined_stereocenter: bool = True,
+    reconnected_metal_layer: bool = True,
+    tautomerism_keto_enol: bool = True,
+    tautomerism_15: bool = True,
+) -> Optional[str]:
```

Comment:
What do you think about passing an argument here to allow supplying arbitrary options?

Reply:
I agree with having a set of pythonic boolean options like you did + an additional `options` argument  that you can just append to the other options if it is not None (like in rdkit)

It's a great combination between your implementation, making popular options more readable/usable, and what Manu suggested in another comment. so that all the options are available if you want to.

Reply:
Yeah, better allowing arbitrary arguments than making a new PR whenever we'll need it.

We can add more boolean flags in the future if we end up using always the same arbitrary option not included as a flag (unlikely to happen, but we never know!).

Reply:
+1